modules/openapi-generator/src/test/java/org/openapitools/codegen/typescript/fetch/TypeScriptFetchModelTest.java

```diff
@@ -384,10 +384,12 @@ public void enumArrayModelTest() {
         fish.put("name", "Fish");
```

Comment:
this test seems to be identical to `modules/openapi-generator/src/test/java/org/openapitools/codegen/php/PhpModelTest.java` - it seems maybe these files / tests were duplicated at some point. Might be worth moving this up onto a higher level and remove the duplication?